lms/templates/fields/field_textarea.underscore

```diff
@@ -43,5 +43,13 @@
                 <span class="u-field-message-help" id="u-field-message-help-<%- id %>"> <%- message %></span>
             </span>
         <% } %>
+        <% if (mode === 'edit') { %>
+          <div class="profile-about-character-count">
+            <span class="cur-char-count"><%- (maxCharacters - value.length) %></span>
```

Comment:
Nit: I don't like abbreviations, so can this just be ``character-count`` or ``current-character-count``.